--- ./poky/meta/recipes-core/base-files/base-files/fstab	2022-01-21 17:41:21.391303643 +0100
+++ ./meta-sulfur/meta/recipes-core/base-files/base-files/fstab	2022-01-21 17:45:48.367303643 +0100
@@ -6,6 +6,8 @@
 tmpfs                /run                 tmpfs      mode=0755,nodev,nosuid,strictatime 0  0
 tmpfs                /var/volatile        tmpfs      defaults              0  0
 
+/dev/sda1            /root/data           ext4       rw,relatime           0  0
+
 # uncomment this if your device has a SD/MMC/Transflash slot
 #/dev/mmcblk0p1       /media/card          auto       defaults,sync,noauto  0  0
 
